src/components/Header.tsx

```diff
@@ -1,18 +1,32 @@
+"use client";
+
 import Image from "next/image";
 import Logo from "@/assets/Logo.svg";
 import { MenuItem } from "./MenuItem";
 import { Container } from "./container";
 
 export function Header (){
     return(
-        <header className="flex justify-between items-center w-full h-20 px-4 bg-black md:px-8">
-            <Image src={Logo} alt={Logo} />
+        <header className="flex justify-between items-center p-4 bg-black mx-[10px] md:mx-[122px] mt-[20px] md:mt-[50px]">
+            <div className=" flex items-center">
+            <Image src={Logo} alt="Logo" />
+            </div>
             <Container>
                 <div className="talkd-menu p-2 rounded-md">
                     <ul className="flex flex-col md:flex-row items-center gap-3">
-                        <MenuItem name="Home" isSelected/>
-                        {/* <MenuItem name="Pricing"/> */}
-                        <MenuItem name="About" isNotSelected />
+                        <MenuItem name="Home" 
+                        
+                        subItems={[
```

Comment:
O que seria isso? Se isso for para poder ter o menu "burger", não podemos implementar assim, isso abre precedente para não termos menus com subitens na landing page nunca.

O que eu faria: implementaria 2 componentes de menu que podem ser carregados dependendo do tamanho da tela usando media query.